src/libraries/System.Text.RegularExpressions/tests/Regex.Cache.Tests.cs

```diff
@@ -137,8 +137,9 @@ void Remove(int n)
 
         private int GetCachedItemsNum()
         {
-            return (int)typeof(Regex)
-                .GetField("s_cacheCount", BindingFlags.NonPublic | BindingFlags.Static)
+            return (int)typeof(Regex).Assembly
+                .GetType("System.Text.RegularExpressions.RegexCache")
+                .GetField("s_cacheCount", BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Static)
```

Comment:
Why add `BindingFlags.Public`? The field is still private.

Reply:
It's not necessary, I'll remove it.  Was left over from some experimentation.